RadarSDK/RadarRoutes.m

```diff
@@ -42,7 +42,7 @@ - (nullable instancetype)initWithObject:(nullable id)object {
 
     id geodesicObj = dict[@"geodesic"];
     if (geodesicObj) {
-        geodesic = [[RadarRouteDistance alloc] initWithObject:geodesicObj];
+        geodesic = [[RadarRouteDistance alloc] initWithObject:geodesicObj[@"distance"]];
```

Comment:
@kochis We probably want to guard against `geodesicObj[@"distance"]` being `nil` since we don't  guard for that in `RadarRouteDistance` `initWithObject:`.

```
id geodesicDistanceObj = geodesicObj[@"distance"];
if (geodesicDistanceObj) {
  geodesic = [[RadarRouteDistance alloc] initWithObject:geodesicDistanceObj];
}
```

A little excessive but want to make sure the deserialization code never crashes.